TripalCultivate/TripalCultivate-Theme
# Color picker

This PR will provide a color picker window to assist user enter a hex color value into the tile background color field.

![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/e4def431-0463-4eaf-ad24-7529eb469e4a)

A hook_form_alter() is used to attach the color picker JS library and target the form to add tile content block.

To test:
1. Access my site/admin/content and click Blocks tab
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/b5e95225-64ad-441e-bef1-11334cbe0358)
2. Click +Add content block button. In the Add content block, click Tripal Cultivate Theme Tile Block to load content entry form.
3. Scroll down to the TIle Background Colour Field.

Color picker will check for invalid hex value:
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/29b57781-f7cc-4de9-8177-51edc8c4d795)
